css/carousel.css

```diff
@@ -0,0 +1,141 @@
+.carousel {
+    margin-bottom: 100px
+  }
+  
+  .carousel-container {
+    width: 96%;
+    height: 400px;
+    margin: 0 auto;
+    position: relative;
+    overflow: hidden;
+    display: flex;
+    border-radius: 10px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    transition: box-shadow 0.3s;
+  }
+  
+  .carousel-container:hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+    animation: size-up 200ms forwards;
+  }
+  
+  @keyframes size-up {
+    100% {
+      transform: scale(1.005);
+    }
+  }
+  
+  #previous {
+    position: absolute;
```

Comment:
Could have reused almost all the styles between both carousel buttons by using a class for both and only define `left` or `right` property for `previous` & `next`.